packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -65,6 +65,28 @@ export class WorkflowWrapper {
 		const updatedNodeMap = new Map<string, WorkflowNode<any>>(updatedWF.nodes.map((n) => [n.id, n]));
 		const updatedEdgeMap = new Map<string, WorkflowEdge>(updatedWF.edges.map((e) => [e.id, e]));
 
+		if (pendingSave || isDragging) {
```

Comment:
Why not creating a single boolean test with a more appropriate names, or a comment explaining why this test is happening?

Reply:
will do